### PR TITLE
Version corrections left over from t4d8 etc.

### PR DESCRIPTION
**Pull request by dsenalik**
1. I did a grep for t4d8 and 9.x and updated these
1. I changed \`pwd\` to $(pwd) because this is preferred, and less likely to be typed in incorrectly - reference http://mywiki.wooledge.org/BashFAQ/082
1. Occasional typo
1. Oh, and I alphabetized the list and added myself 😇

### CODE REVIEW

**Comment by dsenalik** [comment]
There is one more 9.x that I did not change, 

`docs/conf.py` line 37

`release = u'9.x-4.x'`

because I don't know if changing this will break anything, so I left it alone. 🙈

File: docs/install/requirements.rst [hunk 1]
```diff
@@ -24,19 +24,19 @@ Drupal      9.2.x        9.3.x          9.4.x          9.5.x          10.0.x
...
-   :target: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8_D9_5x.yml
-.. |PHP8.1D9.3.x| image:: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_3x.yml/badge.svg
-   :target: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_3x.yml
-.. |PHP8.1D9.4.x| image:: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_4x.yml/badge.svg
-   :target: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_4x.yml
-.. |PHP8.1D9.5.x| image:: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_5x.yml/badge.svg
-   :target: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D9_5x.yml
-.. |PHP8.1D10.0.x| image:: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D10_0x.yml/badge.svg
-   :target: https://github.com/tripal/t4d8/actions/workflows/MAIN-phpunit-php8.1_D10_0x.yml
+.. |PHP8D9.2.x| image:: https://github.com/tripal/tripal/actions/workflows/MAIN-phpunit-php8_D9_2x.yml/badge.svg
```
Comment (laceysanderson) [inline docs/install/requirements.rst:27]: These links don't work... and the same ones are used in the tripal/tripal 4.x readme 😮 We'll need to find the new versions. I see these yml in the tripal/tripal repo with the same name but did not immediately see the run workflows 🤔

**Review by laceysanderson — COMMENTED** [review]
This all looks good except the one comment about broken workflow links... 

Regarding that last instance of `9.x-4.x` I don't think that will break anything so feel free to change it as well.

File: docs/install/requirements.rst (hunk 1 above)
Comment (dsenalik) [reply docs/install/requirements.rst:27]: I think the links are correct, it is just that these workflows have never been run on the 4.x branch.
It looks like in t4d8 they run on the main branch when a pull is merged?
Some badges work e.g. https://github.com/tripal/tripal/actions/workflows/ALL-phpunit.yml/badge.svg
I don't know enough about this to know what may be different or wrong 🤪

Comment (dsenalik) [reply docs/install/requirements.rst:27]: Also, for the release version I did this
`release = u'9.x-4.x'`
became
`release = u'4.x.alpha.1'`
to be in sync with tripal itself. Is this reasonable?

**Comment by dsenalik** [comment]
Linking this Tripal 4 issue, this may be the problem
https://github.com/tripal/tripal/issues/1496

**Comment by dsenalik** [comment]
Badge issue resolved with merged https://github.com/tripal/tripal/issues/1496

**Comment by laceysanderson** [comment]
Great, thanks! We'll want to make the same badge changes in the tripal/tripal repo for the readme

**Comment by dsenalik** [comment]
Pull request for these changes in tripal/tripal repo is https://github.com/tripal/tripal/pull/1499

**Pull request merged**